src/components/connections/Connection.ts

```diff
@@ -81,22 +82,24 @@ export class InfluxDBTreeDataProvider
     }>(InfluxDBConectionsKey);
     const ConnectionNodes = [];
     if (connections) {
+      const activeID = Status.Current?.id;
       for (const id of Object.keys(connections)) {
+        let active = false;
+        if (
+          (activeID !== undefined && activeID === id) ||
```

Comment:
```suggestion
       activeID === id || connections[id].isDefault
```
the === undefined seems redundant here